src/store/state.ts

```diff
@@ -49,6 +49,8 @@ const initialState: RootState = {
   experiments: {},
   watsNewModalVersion: '',
   enabledChains: {},
+  nftAssets: [],
+  starredNFTs: [],
```

Comment:
I left a comment about not requiring a separate state for starred NFTs and just using the property in `nftAssets`. You can again use a getter if you really need it in a list like this